src/components/download_buttons/DownloadButtons.module.scss

```diff
@@ -1,35 +1,87 @@
 @import '../../utils/variables';
 
 .downloadBtns {
-  display: flex;
-  margin: 2em 0;
-  justify-content: space-between;
-  max-width: 600px;
-  flex-wrap: wrap;
-  & a {
-    display: flex;
-    width: 48%;
-    max-width: 300px;
-    max-height: 90px;
-  }
-  & img {
-    max-width: 100%;
-    width: 90%;
-  }
-}
-
-@media (max-width: $small) {
-  .downloadBtns {
+    margin: 2em 0;
+    height: 90px;
     & a {
-      max-height: 50px;
+        &:nth-child(1) {
+            margin-right: 50px;
+        }
+    }
+    & img {
+        height: 100%;
+        object-fit: cover;
+        object-position: left;
     }
-  }
 }
 
+@media (max-width: 1210px) {
+    .downloadBtns {
+        height: 70px;
+    }
+}
+@media (max-width: 1030px) {
+    .downloadBtns {
+        height: 60px;
+        width: 120%;
+    }
+}
 @media (max-width: $medium) {
-  .downloadBtns {
-    & a {
-      max-height: 70px;
+}
+@media (max-width: $small) {
+    .downloadBtns {
+        width: 100%;
+        height: 50px;
+    }
+}
+@media (max-width: 410px) {
+    .downloadBtns {
+        height: 40px;
```

Comment:
I'm also a big fan of nesting all the things so I would say these media queries should go in the .downloadBtn block. Example:
```
.profile-pic {
  float: left;
  width: 250px;
  @media screen and (max-width: $break-small) {
    width: 100px;
    float: none;
  }
```
which compiles to:
```
.profile-pic {
  float: left;
  width: 250px;
}
@media screen and (max-width: 320px) {
  .profile-pic {
    width: 100px;
    float: none;
  }
}
```
http://thesassway.com/intermediate/responsive-web-design-in-sass-using-media-queries-in-sass-32